frontend/src/components/main-page/groups-tree-page/common/dependencies/visualization/graph-hops.ts

```diff
@@ -105,35 +105,35 @@ export function buildHopsGetterFn(
         );
       }
 
-      for (const singleProducedEvent of singleService.producedEvents) {
+      for (const singleProducedEvent of singleService.publishedEvents) {
         if (alreadyVisitedNodes.has(singleProducedEvent)) {
           continue;
         }
         hopsMap.set(singleProducedEvent, APIOrEventDepth);
 
         addMultipleItemsToSet(
-          singleProducedEvent.consumedBy,
+          singleProducedEvent.subscribedBy,
           potentialServicesToVisitInNextIteration,
         );
       }
 
-      for (const singleConsumedEvent of singleService.consumedEvents) {
+      for (const singleConsumedEvent of singleService.subscribedEvents) {
         if (alreadyVisitedNodes.has(singleConsumedEvent)) {
           continue;
         }
         hopsMap.set(singleConsumedEvent, APIOrEventDepth);
 
         addMultipleItemsToSet(
-          singleConsumedEvent.producedBy,
+          singleConsumedEvent.publishedBy,
           potentialServicesToVisitInNextIteration,
         );
       }
```

Comment:
Could you rename `singleProducedEvent` to `singlePublishedEvent`? And `singleConsumedEvent` to `singleSubscribedEvent`?

Reply:
Addressed in [380ba05](https://github.com/osrgroup/msadoc/pull/75/commits/380ba05360e0fcb697ce53874c65ca76dfba6f0e)